doc/Language/classtut.pod6

```diff
@@ -5,86 +5,60 @@
 =SUBTITLE A tutorial about creating and using classes in Raku
 
 X<|OOP>
+Raku provides a rich built-in syntax for defining and using classes. It makes writing classes
+expressive and short for most cases, but also provides mechanisms to cover
+the rare corner cases.
 
-Raku has a rich built-in syntax for defining and using classes.
+=head1 A quick overview
 
-A default constructor allows the setting of attributes for the created
-object:
+Let's start with an example to give an overview:
 
 =begin code
-class Point {
-    has Int $.x;
-    has Int $.y;
-}
-
 class Rectangle {
-    has Point $.lower;
-    has Point $.upper;
+    has Int $.a = 1;
+    has Int $.b = 1;
+
+    submethod TWEAK(:$log) {
+        say "Constructing Rectangle ($!a|$!b)" if $log;
+    }
 
     method area(--> Int) {
-        ($!upper.x - $!lower.x) * ( $!upper.y - $!lower.y);
+        return $!a * $!b;
```

Comment:
OK, this is a bit simpler, and avoids using two examples to illustrate a simple concept. Still, it might be even simpler if you didn't use `TWEAK`, which is an additional concept that's not explained until later. It's not reallly needed either,

Reply:
I just removed TWEAK. I would have liked to show `TWEAK` before `new`, but I'll leave that for a next iteration. (There will be people that skim this page to see how to use a constructor and stop when they see the `new` example and go with that. We should make sure that those people will see `TWEAK` first. This will save those people and us some trouble.)